lib/pure/terminal.nim

```diff
@@ -60,6 +60,22 @@ when defined(windows):
     lpConsoleScreenBufferInfo: ptr CONSOLE_SCREEN_BUFFER_INFO): WINBOOL{.stdcall,
     dynlib: "kernel32", importc: "GetConsoleScreenBufferInfo".}
 
+  proc terminalWidth*(h: Handle): int =
```

Comment:
For consistency with the posix target, make that

proc terminalWidthIoctl*(fds: openArray[int]): int